llumnix/entrypoints/bladellm/utils.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2024, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from loguru import logger
+
+from blade_llm.service.args import ServingArgs
+from llumnix.arg_utils import LlumnixEntrypointsArgs, EngineManagerArgs
+
+def detect_unsupported_feature(engine_args: ServingArgs) -> None:
+    unsupported_feature = None
+    if engine_args.enable_lora:
+        unsupported_feature = "multi-lora serving"
+    elif not engine_args.disable_prompt_cache:
+        unsupported_feature = "automatic prompt caching"
+    elif engine_args.use_sps:
+        unsupported_feature = "speculative decoding"
+    if unsupported_feature:
+        raise ValueError(f'Unsupported feature: Llumnix does not support "{unsupported_feature}" currently.')
```

Comment:
We should update this like 'Unsupported feature: Llumnix does not support "{unsupported_feature}" of BladeLLM currently.'

Reply:
get